CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,31 @@
+[Code of Conduct](https://github.com/percona/community/blob/main/content/contribute/coc.md)
+
+Percona Community is a community around developing, installing, running, optimizing, learning everything around databases and the needed infrastructure.
+
+## The ABCs
+
+* **A**ssume good intentions.
+* **B**e nice to each other - being civil and respectful goes a long way.
+* **C**riticize code, tools, other types of contributions, and situations - not people and their personalities.
+
+## Rules
+
+We respect all members of the Percona Community - that involves everyone’s contributions in code, text, visuals, and chat. Your contributions might take place in one of our GitHub organizations, on Jira, on the forums, Slack, or elsewhere.
```

Comment:
Community has no access to Slack. Contributing elsewhere means we may never be able to see the contribution.